integration_tests/integration/createLicence.cy.ts

```diff
@@ -16,6 +16,7 @@ context('Create a licence', () => {
     cy.task('stubGetActivePolicyConditions')
     cy.task('stubGetBankHolidays', dates)
     cy.task('stubAddAdditionalCondition')
+    cy.task('stubFeComponents')
```

Comment:
I'll add this to all cypress tests as a follow up - it adds loads of errors/noise to the logs if this is not stubbed out

Reply:
I've included this now